### PR TITLE
Enable Hearthstone client -> Stove reach the main menu

### PR DESCRIPTION
**Pull request by djkaty**
These lovingly crafted changes allow Hearthstone clients to connect, login, exchange data and reach the game's main menu using Hearthstone 4.3.12266. Crafting and disenchanting is confirmed to work, everything else is broken in one way or another.

### CODE REVIEW

File: pegasus/account.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func (s *Session) HandleAccountInfoRequest(req util.GetAccountInfo_Request) *Pac
 		res.LevelEnd = proto.Int32(3)
 		res.CanLoseLevel = proto.Bool(false)
 		return EncodePacket(util.MedalInfo_ID, &res)
-	case util.GetAccountInfo_MEDAL_HISTORY:
+/*	case util.GetAccountInfo_MEDAL_HISTORY:
```
Comment (jleclanche) [inline pegasus/account.go]: No need to comment this stuff out, you can delete it for now.

**Comment by jleclanche** [comment]
gtm. @Mischanix?

**Comment by Mischanix** [comment]
lgtm

**Comment by djkaty** [comment]
Commented out code removed

**Pull request merged**